src/platform/user/profile/utilities/index.js

```diff
@@ -88,20 +88,20 @@ export function mapRawUserDataToState(json) {
       ? vet360ContactInformation
       : mockContactInformation,
     session,
+    veteranStatus: {
```

Comment:
I shoulda noticed this before: Although setting this here is probably harmless, I think I'd prefer to save setting this until down around line 104. Just setting it to an empty object here will allow line 102 to continue to function as well as indicate that `veteranStatus` is supposed to be an object of some kind.

Reply:
Oh! And unrelated to this PR, but line 87 is what you'll need to change (or change how `isVet360Configured` works) to make Cypress tests work when testing address updates!